### ISSUE TITLE
Opening links from TestDocumentList display an error

### ISSUE THREAD

**Issue by oanat** [xwikisas/application-googleapps#48]
Steps to reproduce:
* Configure XWiki 11.10.5 with GA 2.4.5
* Login with a Google account
* Go to http://<server>/xwiki/bin/view/GoogleApps/TestDocumentList
* Click on any link from the list of documents displayed

Result:
```
{
 "error": {
  "errors": [
   {
    "domain": "usageLimits",
    "reason": "dailyLimitExceededUnreg",
    "message": "Daily Limit for Unauthenticated Use Exceeded. Continued use requires signup.",
    "extendedHelp": "https://code.google.com/apis/console"
   }
  ],
  "code": 403,
  "message": "Daily Limit for Unauthenticated Use Exceeded. Continued use requires signup."
 }
}
```

**Comment by polx** [comment]
I was able to reproduce this and can say that this is a change on Google's side.

**Comment by polx** [comment]
Solved and released in 2.4.6. Please re-open if still encountering.
